LICENSE

```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
+
+Copyright (c) 2018, Digital Bazaar, Inc.
```

Comment:
Is this the proper license?  If so, please bump the date.

This is the license we use in other w3c-dvcg repos: https://github.com/w3c-dvcg/security-vocab/blob/master/LICENSE.md

Reply:
I copied this from somewhere (probably bedrock) I will update it with the latest license in a second.

Reply:
This was addressed in another branch in this commit: https://github.com/w3c-dvcg/http-signatures-test-suite/pull/5/commits/4ce66659726981b6ae0b0a82976bf9892d2195c7

This PR is months out of date and all-tests is the only current one.